praw/models/reddit/subreddit.py

```diff
@@ -122,7 +122,7 @@ def banned(self):
 
         .. code-block:: python
 
-           reddit.subreddit('SUBREDDIT').banned.add('NAME', reason='a reason')
+           reddit.subreddit('SUBREDDIT').banned.add('NAME', ban_reason='a reason')
```

Comment:
This line is failing the style check as it's now just a bit too long. Could you put the arguments on their own line, or maybe change `a reason` with `...`?